compiler/luci/pass/src/FuseGRUPass.test.cpp

```diff
@@ -0,0 +1,418 @@
+    _split_out_node_5 = _body_graph->nodes()->create<luci::CircleSplitOut>();
+    _split_out_node_6 = _body_graph->nodes()->create<luci::CircleSplitOut>();
+
+    _reshape_node = _body_graph->nodes()->create<luci::CircleReshape>();
+
+    auto graph_input_cond_graph = _cond_graph->inputs()->create();
+    _cond_input_node = _cond_graph->nodes()->create<luci::CircleInput>();
+    _cond_input_node->index(graph_input_cond_graph->index());
+
+    auto graph_output_cond_graph = _cond_graph->outputs()->create();
+    _cond_output_node = _cond_graph->nodes()->create<luci::CircleOutput>();
+    _cond_output_node->index(graph_output_cond_graph->index());
+
+    auto graph_input_body_graph_1 = _body_graph->inputs()->create();
+    _body_input_node_1 = _body_graph->nodes()->create<luci::CircleInput>();
+    _body_input_node_1->index(graph_input_body_graph_1->index());
+
+    auto graph_input_body_graph_2 = _body_graph->inputs()->create();
+    _body_input_node_2 = _body_graph->nodes()->create<luci::CircleInput>();
+    _body_input_node_2->index(graph_input_body_graph_2->index());
+
+    auto graph_input_body_graph_3 = _body_graph->inputs()->create();
+    _body_input_node_3 = _body_graph->nodes()->create<luci::CircleInput>();
+    _body_input_node_3->index(graph_input_body_graph_3->index());
+
+    auto graph_input_body_graph_4 = _body_graph->inputs()->create();
+    _body_input_node_4 = _body_graph->nodes()->create<luci::CircleInput>();
+    _body_input_node_4->index(graph_input_body_graph_4->index());
+
+    auto graph_input_body_graph_5 = _body_graph->inputs()->create();
+    _body_input_node_5 = _body_graph->nodes()->create<luci::CircleInput>();
+    _body_input_node_5->index(graph_input_body_graph_5->index());
+
+    auto graph_output_body_graph_1 = _body_graph->outputs()->create();
+    _body_output_node_1 = _body_graph->nodes()->create<luci::CircleOutput>();
+    _body_output_node_1->index(graph_output_body_graph_1->index());
+
+    auto graph_output_body_graph_2 = _body_graph->outputs()->create();
+    _body_output_node_2 = _body_graph->nodes()->create<luci::CircleOutput>();
+    _body_output_node_2->index(graph_output_body_graph_2->index());
+
+    auto graph_output_body_graph_3 = _body_graph->outputs()->create();
+    _body_output_node_3 = _body_graph->nodes()->create<luci::CircleOutput>();
+    _body_output_node_3->index(graph_output_body_graph_3->index());
+
+    auto graph_output_body_graph_4 = _body_graph->outputs()->create();
+    _body_output_node_4 = _body_graph->nodes()->create<luci::CircleOutput>();
+    _body_output_node_4->index(graph_output_body_graph_4->index());
+
+    auto graph_output_body_graph_5 = _body_graph->outputs()->create();
+    _body_output_node_5 = _body_graph->nodes()->create<luci::CircleOutput>();
+    _body_output_node_5->index(graph_output_body_graph_5->index());
+  }
+
+  void invalid_less_const_type() { _less_const_node->dtype(loco::DataType::S16); }
+
+protected:
+  luci::CircleWhile *_while_node;
```

Comment:
```suggestion
  luci::CircleWhile *_while_node = nullptr;
```
and others in below too.

Reply:
updated as suggested.